web/tests/functional/products/test_products.py

```diff
@@ -168,6 +168,8 @@ def test_get_product_data(self):
                          os.path.basename(self.test_workspace),
                          "The displayed name must == the default value, as "
                          "we didn't specify a custom displayed name.")
+        confidentiality = pr_conf.confidentiality
```

Comment:
Please extend the gui test cases too: https://github.com/Ericsson/codechecker/blob/master/web/server/vue-cli/e2e/specs/products.js#L92